<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ethernet</class>
 <widget class="QWidget" name="Ethernet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Ethernet Extension</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="ethernet_hostname_label">
             <property name="text">
              <string>Hostname:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ethernet_hostname">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Up to 15 chars, leave empty for default</string>
             </property>
             <property name="maxLength">
              <number>32</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Port:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="ethernet_port">
             <property name="maximum">
              <number>65535</number>
             </property>
             <property name="value">
              <number>4223</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Connection:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="ethernet_connection">
             <item>
              <property name="text">
               <string>DHCP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Static IP</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="ethernet_ip_label">
             <property name="text">
              <string>IP:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="ethernet_ip_spacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QSpinBox" name="ethernet_ip4">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ethernet_dot1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="ethernet_ip3">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ethernet_dot2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="ethernet_ip2">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ethernet_dot3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="ethernet_ip1">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="ethernet_sub_label">
             <property name="text">
              <string>Subnet Mask:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="ethernet_sub_spacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QSpinBox" name="ethernet_sub4">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ethernet_dot4">
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="ethernet_sub3">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ethernet_dot5">
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="ethernet_sub2">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ethernet_dot6">
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="ethernet_sub1">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="ethernet_gw_label">
             <property name="text">
              <string>Gateway:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="ethernet_gw_spacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QSpinBox" name="ethernet_gw4">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ethernet_dot7">
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="ethernet_gw3">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ethernet_dot8">
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="ethernet_gw2">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ethernet_dot9">
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="ethernet_gw1">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="mac_layout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>MAC Address:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="ethernet_count_rx">
             <property name="text">
              <string>RX: 0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="Line" name="line_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="ethernet_count_tx">
             <property name="text">
              <string>TX: 0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="ethernet_save">
             <property name="text">
              <string>Save Ethernet Configuration</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
